### PR TITLE
Added statistic Manager and Debate statistic page. Working on question statistic page

### PR DESCRIPTION
**Pull request by Naludrag**
working on #20.

### CODE REVIEW

File: src/app/pages/admin/closedquestion-stats/closedquestion-stats.page.spec.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    }).compileComponents();
+
+    fixture = TestBed.createComponent(ClosedQuestionStatsPage);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  }));
+
+  /*it('should create', () => {
+    expect(component).toBeTruthy();
+  });*/
```
Comment (BecaudA) [inline src/app/pages/admin/closedquestion-stats/closedquestion-stats.page.spec.ts:23]: Remove unused code.

File: src/app/pages/admin/question-admin/question-admin.page.spec.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    }).compileComponents();
+
+    fixture = TestBed.createComponent(QuestionAdminPage);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  }));
+
+  /*it('should create', () => {
+    expect(component).toBeTruthy();
+  });*/
```
Comment (BecaudA) [inline src/app/pages/admin/question-admin/question-admin.page.spec.ts:23]: Remove unused code.

File: src/app/services/statistic.service.spec.ts [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+        })
+
+    });
+
+    it('should be created', () => {
+        const service: StatService = TestBed.get(StatService);
+        expect(service).toBeTruthy();
+    });
+
+    // TODO: Add tests
```
Comment (BecaudA) [inline src/app/services/statistic.service.spec.ts:22]: You must implement of statistic service.

File: src/app/pages/admin/closedquestion-stats/closedquestion-stats.page.spec.ts (hunk 1 above)
Comment (Naludrag) [reply src/app/pages/admin/closedquestion-stats/closedquestion-stats.page.spec.ts]: Removed in new commit

File: src/app/pages/admin/question-admin/question-admin.page.spec.ts (hunk 2 above)
Comment (Naludrag) [reply src/app/pages/admin/question-admin/question-admin.page.spec.ts]: Removed in the last commit

File: src/app/services/statistic.service.spec.ts (hunk 3 above)
Comment (Naludrag) [reply src/app/services/statistic.service.spec.ts]: I have no inspiration to add more tests so I removed the TODO

**Pull request merged**